aspnetcore/fundamentals/minimal-apis/bindStreamPipeReader/7.0-samples/PipeStreamToBackgroundQueue/Program.cs

```diff
@@ -0,0 +1,31 @@
+using System.Threading.Channels;
+using BackgroundQueueService;
+var builder = WebApplication.CreateBuilder(args);
+// Create a channel to send data to the background queue.
+builder.Services.AddSingleton<Channel<Memory<byte>>>((_) => Channel.CreateBounded<Memory<byte>>(100));
```

Comment:
I messed up, this should be `ReadOnlyMemory<byte>` 😄. We're close!

Reply:
Alright 🥂